examples/svelte-apollo/src/graphql.js

```diff
@@ -0,0 +1,8 @@
+import { gql } from '@apollo/client'
+export const AUTHOR_LIST = gql`
+  query {
+    author(order_by: [{ name: asc }]) {
+      name
+    }
+  }
+`
```

Comment:
Some GraphQL variables are specified here and some inside `.svelte` files. I think we should be consistent on where to specify GraphQL variables.

Reply:
All GraphQL Variables are in only ``.svelte`` files only .